interfaces/devdoc/threadpool_requirements.md

```diff
@@ -112,45 +112,43 @@ MOCKABLE_FUNCTION(, THREADPOOL_WORK_ITEM_HANDLE, threadpool_create_work_item, TH
 
 **NON_THREADPOOL_05_002: [** If `work_function` is `NULL`, `threadpool_create_work_item` shall fail and return a `NULL` value. **]**
 
-**NON_THREADPOOL_05_003: [** Otherwise `threadpool_create_work_item` shall allocate a context of type `THREADPOOL_WORK_ITEM_HANDLE` to save threadpool related variables. **]**
+**NON_THREADPOOL_05_003: [** Otherwise `threadpool_create_work_item` shall allocate a context of type `THANDLE(THREADPOOL_WORK_ITEM)` to save threadpool related variables. **]**
 
 **NON_THREADPOOL_05_004: [** If any error occurs, `threadpool_create_work_item` shall fail and return a `NULL` value. **]**
 
-**NON_THREADPOOL_05_005: [** `threadpool_create_work_item` shall create an individual work item context of type `THREADPOOL_WORK_ITEM_HANDLE` **]**
+**NON_THREADPOOL_05_005: [** `threadpool_create_work_item` shall create an individual work item context of type `THANDLE(THREADPOOL_WORK_ITEM)` **]**
```

Comment:
> context of type `THANDLE(THREADPOOL_WORK_ITEM) [](http://example.com/codeflow?start=95&length=47)

wasn't it already created in line 115 of this review?

Reply:
Fixed.